portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,25 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+  console.log("Helloooo");
 }
+
+function randomImage(){
+    const img_container = document.getElementById('img-container');
+    var imgAr = new Array();
+    imgAr[0] = 'images/city-birdeye.jpg';
+    imgAr[1] = 'images/night-city.jpg';
+    imgAr[2] = 'images/nightcity-streeta2.jpg';
+    var num = Math.floor( Math.random() * imgAr.length );
+
+    img_container.src = imgAr[num];
+    console.log(num);
+}
+
+// function getRandomImage(imgAr, path) {
+//     path = path || 'images/'; // default path here
+//     var num = Math.floor( Math.random() * imgAr.length );
+//     var img = new imgArray(4)imgAr[ num ];
+//     var imgStr = '<img src="' + path + img + '" alt = "">';
+//     document.write(imgStr); document.close();
+// }
```

Comment:
Reposting this from #2.

It looks like this an alternative attempt to `randomImage()`? If so, you can delete this. You can always come back to it by viewing this file at the commit before it was deleted.